packaging/pre_build_script.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+set -eux
+
+echo "This script is run before building torchao binaries"
+
+pip install setuptools wheel twine auditwheel
+
+# NB: This will make the nightly wheel compatible with the latest stable torch,
```

Comment:
Yeah I think the correct release strategy might be
1. Nightly ao with nightly torch
2. Release ao with release torch

So at least to merge this let's use the nightly torch versions

Reply:
Then the fix is very simple, we just need to remove the step to install torch here.  The build workflow has already installed the correct torch version, i.e. nightly/release torch for nightly/release ao respectively